packages/terra-form-textarea/src/terra-dev-site/doc/example/Invalid.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Textarea from 'terra-form-textarea/lib/Textarea';
+
+class InvalidExample extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      area: 'Leaving this as blank makes this input invalid.',
+    };
+    this.handleAreaChange = this.handleAreaChange.bind(this);
+  }
+
+  handleAreaChange(event) {
+    this.setState({ area: event.target.value });
+  }
+
+  render() {
+    return (
+      <Textarea
+        value={this.state.area}
+        isInvalid={this.state.area === ''}
+        onChange={this.handleAreaChange}
+        size="small"
```

Comment:
Can we start this example as invalid? We can add `Leaving this as blank makes this input invalid.` to the example description.

Reply:
8502c90